### ISSUE TITLE
Use GitHub app for governance votes

### ISSUE THREAD

**Issue by jezdez** [conda/governance#195]
### Checklist

- [x] I added a descriptive title
- [x] I searched open requests and couldn't find a duplicate

### What is the idea?

I stumbled over https://github.com/cncf/gitvote and thought this might be a great way to facilitate votes according to the governance policy

### Why is this needed?

The existing ad-hoc vote messages in PRs and issues are inconsistent and have a problem of record-keeping, not the least represent a barrier of contribution from steering council members.

### What should happen?

Trial https://github.com/cncf/gitvote and see if it would fit with our voting items.

**Comment by jaimergp** [comment]
Love it! Let's add it!

**Comment by mariusvniekerk** [comment]
Looks neat. I assume we can restrict who can vote by using teams

**Comment by jaimergp** [comment]
> The way it computes passing votes does not match our governance policy.

Can you elaborate? Is it about [this part of the code](https://github.com/cncf/gitvote/blob/1778be36dcb3c137a8ee30003842111794e685b8/src/results.rs#L192-L198)?

**Comment by beckermr** [comment]
Yep. Read the governance docs on how we count vote totals and compare. We have quorum thresholds, special rules, etc.
